helix-term/src/commands.rs

```diff
@@ -5392,7 +5395,32 @@ fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
             textobject(cx.editor);
             cx.editor.last_motion = Some(Motion(Box::new(textobject)));
         }
-    })
+    });
+
+    if let Some((title, abbrev)) = match objtype {
+        textobject::TextObject::Inside => Some(("Match inside", "mi")),
+        textobject::TextObject::Around => Some(("Match around", "ma")),
+        textobject::TextObject::Movement => None,
+    } {
+        let help_text = [
+            ("w", "Word"),
+            ("W", "WORD"),
+            ("c", "Class (tree-sitter)"),
+            ("f", "Function (tree-sitter)"),
+            ("p", "Parameter (tree-sitter)"),
+            ("m", "Matching delimiter under cursor"),
+            (" ", "... or any ascii character acting as a pair"),
```

Comment:
We should remove this later as well after implementing more pairs so I think not worth having a separate manual help text.

Reply:
It does work with any alphanumeric though. I tested it when I was working on it.

Reply:
Yes it does, but it is not useful. Which is what I am saying.

Reply:
See latest comments in the main thread, I disagree on this point.

Reply:
I like this annotation, keep it. I'd remove the `ascii` part and just say or "or any other character"